docs/getting-started-scalardl-auditor.md

```diff
@@ -0,0 +1,545 @@
+   ```
+   If the Auditor Pods are deployed properly, you can see the STATUS are `Running`.  
+
+1. Check the Auditor Services are deployed.
+   ```console
+   $ kubectl get svc
+   NAME                             TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)                           AGE
+   kubernetes                       ClusterIP   10.96.0.1       <none>        443/TCP                           25m
+   scalardl-auditor-envoy           NodePort    10.101.34.229   <none>        40051:30140/TCP,40052:30599/TCP   3m32s
+   scalardl-auditor-envoy-metrics   ClusterIP   10.97.47.11     <none>        9001/TCP                          3m32s
+   scalardl-auditor-headless        ClusterIP   None            <none>        50051/TCP,50053/TCP,50052/TCP     3m32s
+   scalardl-auditor-metrics         ClusterIP   10.103.88.93    <none>        8080/TCP                          3m32s
+   scalardl-ledger-envoy            NodePort    10.97.160.80    <none>        50051:32468/TCP,50052:30626/TCP   19m
+   scalardl-ledger-envoy-metrics    ClusterIP   10.97.83.78     <none>        9001/TCP                          19m
+   scalardl-ledger-headless         ClusterIP   None            <none>        50051/TCP,50053/TCP,50052/TCP     19m
+   scalardl-ledger-metrics          ClusterIP   10.111.170.92   <none>        8080/TCP                          19m
+   ```
+   If the Auditor Services are deployed properly, you can see a private IP address in the CLUSTER-IP column. (Note: `scalardl-auditor-headless` has no CLUSTER-IP.)  
+
+1. (Optional) If you set `LoadBalancer` to `envoy.service.type` in the `scalardl-auditor-custom-values.yaml`, you can access Auditor from 127.0.0.1.  
+   To expose the `scalardl-auditor-envoy` service as your local `127.0.0.1:40051` and `127.0.0.1:40052`, open another terminal, and run the `minikube tunnel` command.
+   ```console
+   $ minikube tunnel
+   ```
+   After running the `minikube tunnel` command, you can see the  EXTERNAL-IP of the `scalardl-auditor-envoy` as  `127.0.0.1`.
+   ```console
+   $ kubectl get svc scalardl-auditor-envoy
+   NAME                     TYPE           CLUSTER-IP      EXTERNAL-IP   PORT(S)                           AGE
+   scalardl-auditor-envoy   LoadBalancer   10.101.34.229   127.0.0.1     40051:30140/TCP,40052:30599/TCP   5m28s
+   ```
+
+## Step 6. Start Client container
+
+In this section, we will create a Client container to run a sample contract of Scalar DL.  
+We will use certificate files in the Client container. So, mount ~/scalardl-test/certs to the Client container.  
+
+1. Start the Client container on the Docker Network `minikube`.
+   ```console
+   $ docker run -d --name scalardl-client --hostname scalardl-client -v ~/scalardl-test/certs:/certs --network minikube --entrypoint sleep ubuntu:20.04 inf
+   ```
+
+1. Check the Client container is running.
+   ```console
+   $ docker ps -f name=scalardl-client
+   ```
+
+## Step 7. Run Scalar DL sample contract in the Client container
```

Comment:
Ditto

Reply:
I think this section is almost the same between Ledger and Ledger+Auditor.  
However, there are additional steps in the Ledger+Auditor document.  

For example, we need to create property files of Ledger and Auditor as client, to register each certificate files.  
And, there are some steps that register each certificate files between Ledger and Auditor.  

Also, I describe the result of Contract Execution Request and Validation Request as samples to make behavior of Scalar DL more understandable.  
And, the result of each request is difference between Ledger and Ledger+Auditor.  

So, I think it is better that we don't separate this section.  
What do you think about the above policy?

Reply:
@kota2and3kan san,
I am not sure about this policy.
I think points 1 to 8 in this step are identical.

So, you can confirm with @feeblefakie san.

Reply:
I think some redundancy is fine if it is easy to understand/follow for users.

Reply:
@feeblefakie 
Thank you for your comment!
In this PR, I will keep this redundancy for users can use by copy and paste.

@scalar-boney 
According to the discussion, I will keep this redundancy in this PR.
Because I want to make it users can use by copy and paste as much as possible.

However, I agree that the redundancy causes some maintenance issues.
So, I will consider making this doc easier for users to use and easier for developers to maintain in the future.
Thank you for your suggestion!